partiql-tests-data/success/syntax/primitives/union-except-intersect.ion

```diff
@@ -191,33 +191,69 @@
         }
     },
     {
-        name: "nested bag op with ORDER BY LIMIT OFFSET",
+        name: "nested SQL set op with ORDER BY LIMIT OFFSET",
         statement: '''
             (
                 (SELECT a1 FROM b1 ORDER BY c1 LIMIT d1 OFFSET e1)
                 UNION DISTINCT
                 (SELECT a2 FROM b2 ORDER BY c2 LIMIT d2 OFFSET e2)
             )
-            OUTER UNION ALL
+            UNION ALL
             (SELECT a3 FROM b3 ORDER BY c3 LIMIT d3 OFFSET e3)
             ORDER BY c4 LIMIT d4 OFFSET e4''',
         assert: {
             result: SyntaxSuccess
         }
     },
     {
-        name: "deep nested bag ops with ORDER BY LIMIT OFFSET",
+        name: "bag ops with ORDER BY LIMIT OFFSET in children",
         statement: '''
-            (a UNION b)
-            INTERSECT
-            (
-                c EXCEPT
-                d UNION ALL (e INTERSECT f ORDER BY g LIMIT h OFFSET i)
-            )
-            EXCEPT j
-            ORDER BY k LIMIT l OFFSET m''',
+            (SELECT a1 FROM b1 ORDER BY c1 LIMIT d1 OFFSET e1)
+            OUTER UNION
+            (SELECT a2 FROM b2 ORDER BY c2 LIMIT d2 OFFSET e2)
+            OUTER INTERSECT ALL
+            (SELECT a3 FROM b3 ORDER BY c3 LIMIT d3 OFFSET e3)
+            OUTER EXCEPT DISTINCT
+            (SELECT a4 FROM b4 ORDER BY c4 LIMIT d4 OFFSET e4)
+            ''',
         assert: {
             result: SyntaxSuccess
         }
     },
+    {
```

Comment:
Some additional PartiQL bag op tests to not allow for parsing of `OUTER <set op>` with `ORDER BY`, `LIMIT`, or `OFFSET`.